### PR TITLE
Fixed card square

### PR DESCRIPTION
**Pull request by tsivkadi**
before
image

after
image

### CODE REVIEW

File: src/components/img-title-description/ImgTitleDescription.tsx [hunk 1]
```diff
@@ -3,10 +3,12 @@ import { FC } from 'react'
 import Box from '@mui/material/Box'
 import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
 import { SxProps } from '@mui/system'
+import Avatar from '@mui/material/Avatar'
 
 import { styles } from '~/components/img-title-description/ImgTitleDescription.styles'
 
 interface ImgTitleDescriptionProps {
+  component?: string
```
Comment (MickhaelSh) [inline src/components/img-title-description/ImgTitleDescription.tsx:11]: imgComponent if is used for img

File: src/components/img-title-description/ImgTitleDescription.tsx [hunk 2]
```diff
@@ -16,14 +18,19 @@ interface ImgTitleDescriptionProps {
 }
 
 const ImgTitleDescription: FC<ImgTitleDescriptionProps> = ({
+  component,
```
Comment (MickhaelSh) [inline src/components/img-title-description/ImgTitleDescription.tsx:21]: default 'img'

File: src/components/img-title-description/ImgTitleDescription.tsx [hunk 3]
```diff
@@ -16,14 +18,19 @@ interface ImgTitleDescriptionProps {
...
   description,
   style = styles
 }) => {
   return (
     <Box sx={style.root}>
-      <Box alt='info' component='img' src={img} sx={style.img} />
+      {component === 'avatar' ? (
+        <Avatar src={img} sx={style.img} />
+      ) : (
+        <Box alt='info' component='img' src={img} sx={style.img} />
```
Comment (MickhaelSh) [inline src/components/img-title-description/ImgTitleDescription.tsx:32]: component={imgComponent}

File: src/components/img-title-description/ImgTitleDescription.tsx (hunk 1 above)
Comment (tsivkadi) [reply src/components/img-title-description/ImgTitleDescription.tsx]: fixed

**Review by MickhaelSh — APPROVED** [review]

File: src/containers/find-offer/offer-card-square/OfferCardSquare.tsx [hunk 4]
```diff
@@ -103,20 +105,22 @@ const OfferCardSquare: FC<OfferCardSquareProps> = ({
...
-        <TitleWithDescripiton
-          description={t('common.hourSlash')}
-          style={styles.titleWithDescription}
-          title={`${price} ${t('common.uah')}`}
-        />
-        <Box>
-          <AppRatingMobile
-            reviewsCount={author.totalReviews}
-            value={authorAvgRating}
+      <Box>
```
Comment (Yurenko) [inline src/containers/find-offer/offer-card-square/OfferCardSquare.tsx:108]: Why do you wrap these components additional Box? Delete it. This is redundant

File: src/containers/find-offer/offer-card-square/OfferCardSquare.tsx [hunk 5]
```diff
@@ -103,20 +105,22 @@ const OfferCardSquare: FC<OfferCardSquareProps> = ({
...
+        <Box sx={styles.priceContainer}>
+          <TitleWithDescripiton
+            description={t('common.hourSlash')}
+            style={styles.titleWithDescription}
+            title={`${price} ${t('common.uah')}`}
           />
+          <Box>
```
Comment (Yurenko) [inline src/containers/find-offer/offer-card-square/OfferCardSquare.tsx:115]: The same here.

File: src/containers/find-offer/offer-card-square/OfferCardSquare.tsx (hunk 4 above)
Comment (tsivkadi) [reply src/containers/find-offer/offer-card-square/OfferCardSquare.tsx:108]: this is for better card appearance and its responsibility, this block with price and buttons should always be at the bottom regardless of the card size.

Comment (maarekua) [reply src/containers/find-offer/offer-card-square/OfferCardSquare.tsx:108]: I think we should handle it without an additional block.

**Review by Yurenko — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**